### DISCUSSION TITLE
missing config file

### DISCUSSION THREAD

**Discussion by gabomoncio** [RakambdaOrg/FallingTree#344] [Q&A] [answered]
well as the title says i cant find the config file for my server lol

**Comment by Rakambda** [comment, marked as answer]
In the config folder (next to the mods folder), like probably a lot of mods.
Should be called fallingtree.json (if using a not too old version).

Other possibility is that the mod hasn't been loaded at least once on the server side and didn't create the config file.

**Reply by gabomoncio** [reply]
yes i figured the problem was forge thx for the help
